etl/steps/data/garden/un/2022-07-11/un_wpp/un_wpp.countries.json

```diff
@@ -0,0 +1,267 @@
+  "Upper-middle-income countries": "Upper-middle-income countries",
+  "Lower-middle-income countries": "Lower-middle-income countries",
+  "Low-income countries": "Low-income countries",
+  "Africa": "Africa (UN)",
+  "AFRICA": "Africa (UN)",
+  "Burundi": "Burundi",
+  "Comoros": "Comoros",
+  "Djibouti": "Djibouti",
+  "Eritrea": "Eritrea",
+  "Ethiopia": "Ethiopia",
+  "Kenya": "Kenya",
+  "Madagascar": "Madagascar",
+  "Malawi": "Malawi",
+  "Mauritius": "Mauritius",
+  "Mayotte": "Mayotte",
+  "Mozambique": "Mozambique",
+  "R\u00e9union": "Reunion",
+  "Rwanda": "Rwanda",
+  "Seychelles": "Seychelles",
+  "Somalia": "Somalia",
+  "South Sudan": "South Sudan",
+  "Uganda": "Uganda",
+  "United Republic of Tanzania": "Tanzania",
+  "Zambia": "Zambia",
+  "Zimbabwe": "Zimbabwe",
+  "Angola": "Angola",
+  "Cameroon": "Cameroon",
+  "Central African Republic": "Central African Republic",
+  "Chad": "Chad",
+  "Congo": "Congo",
+  "Democratic Republic of the Congo": "Democratic Republic of Congo",
+  "Equatorial Guinea": "Equatorial Guinea",
+  "Gabon": "Gabon",
+  "Sao Tome and Principe": "Sao Tome and Principe",
+  "Algeria": "Algeria",
+  "Egypt": "Egypt",
+  "Libya": "Libya",
+  "Morocco": "Morocco",
+  "Sudan": "Sudan",
+  "Tunisia": "Tunisia",
+  "Western Sahara": "Western Sahara",
+  "Botswana": "Botswana",
+  "Eswatini": "Eswatini",
+  "Lesotho": "Lesotho",
+  "Namibia": "Namibia",
+  "South Africa": "South Africa",
+  "Benin": "Benin",
+  "Burkina Faso": "Burkina Faso",
+  "Cabo Verde": "Cape Verde",
+  "C\u00f4te d'Ivoire": "Cote d'Ivoire",
+  "Gambia": "Gambia",
+  "Ghana": "Ghana",
+  "Guinea": "Guinea",
+  "Guinea-Bissau": "Guinea-Bissau",
+  "Liberia": "Liberia",
+  "Mali": "Mali",
+  "Mauritania": "Mauritania",
+  "Niger": "Niger",
+  "Nigeria": "Nigeria",
+  "Saint Helena": "Saint Helena",
+  "Senegal": "Senegal",
+  "Sierra Leone": "Sierra Leone",
+  "Togo": "Togo",
+  "Asia": "Asia (UN)",
+  "ASIA": "Asia (UN)",
+  "Kazakhstan": "Kazakhstan",
+  "Kyrgyzstan": "Kyrgyzstan",
+  "Tajikistan": "Tajikistan",
+  "Turkmenistan": "Turkmenistan",
+  "Uzbekistan": "Uzbekistan",
+  "China": "China",
+  "China, Hong Kong SAR": "Hong Kong",
+  "China, Macao SAR": "Macao",
+  "China, Taiwan Province of China": "Taiwan",
+  "Dem. People's Republic of Korea": "North Korea",
+  "Japan": "Japan",
+  "Mongolia": "Mongolia",
+  "Republic of Korea": "South Korea",
+  "Afghanistan": "Afghanistan",
+  "Bangladesh": "Bangladesh",
+  "Bhutan": "Bhutan",
+  "India": "India",
+  "Iran (Islamic Republic of)": "Iran",
+  "Maldives": "Maldives",
+  "Nepal": "Nepal",
+  "Pakistan": "Pakistan",
+  "Sri Lanka": "Sri Lanka",
+  "Brunei Darussalam": "Brunei",
+  "Cambodia": "Cambodia",
+  "Indonesia": "Indonesia",
+  "Lao People's Democratic Republic": "Laos",
+  "Malaysia": "Malaysia",
+  "Myanmar": "Myanmar",
+  "Philippines": "Philippines",
+  "Singapore": "Singapore",
+  "Thailand": "Thailand",
+  "Timor-Leste": "East Timor",
+  "Timor": "East Timor",
```

Comment:
I don't think this mapping is correct. You could rename "Timor": "Timor (UN)"  (in principle you could rename it simply "Timor", but given all the confusion we've had, it's probably better to avoid that name).

Reply:
(Unless for some reason in this dataset "Timor" actually meant "East Timor")